app/controllers/admin/finance/statements_controller.rb

```diff
@@ -46,7 +46,7 @@ def statements_query
         contract_period_years:,
         statement_date:,
         fee_type:,
-        order_by: :statement_date,
+        sort: "+year,+month",
```

Comment:
slightly confused by this, we don't allow sorting at all in statements as you mention, why would we add it 🤔

Reply:
We hard code this in to give the statements the correct/matching ordering with ECF; we need to support a `sort` attribute on the query as the admin area also pulls in statements and sorts by `payment_date`